airflow/contrib/hooks/aws_hook.py

```diff
@@ -18,79 +18,38 @@
 # under the License.
 
 """
-This module contains Base AWS Hook
+This module contains Base AWS Hook.
+
+.. seealso::
+    For more information on how to use this hook, take a look at the guide:
+    :ref:`howto/connection:AWSHook`
 """
 
 import configparser
 import logging
 
 import boto3
+from botocore.config import Config
 
 from airflow.exceptions import AirflowException
 from airflow.hooks.base_hook import BaseHook
 
 
-def _parse_s3_config(config_file_name, config_format='boto', profile=None):
```

Comment:
Moved this function after the Class, so the documentation for actual AWS Hook is at the top of the page in Sphinx